parsl/monitoring/monitoring.py

```diff
@@ -305,7 +317,7 @@ def __init__(self,
                  monitoring_hub_address="127.0.0.1",
                  logdir=".",
                  run_id=None,
-                 logging_level=logging.DEBUG,
+                 logging_level=logging.INFO,
```

Comment:
Shouldn't this be `logging_level=logging_level`

Reply:
I think this should be `logging.INFO` -- it is the default logging option for `Hub` class and it is set on line 232.

Reply:
Oh, sorry. I misread this.